src/java/org/apache/ivy/osgi/p2/XMLInputParser.java

```diff
@@ -19,12 +19,11 @@
 
 import java.io.IOException;
 import java.io.InputStream;
-import java.text.ParseException;
 
 import org.xml.sax.SAXException;
 
 public interface XMLInputParser {
 
-    void parse(InputStream in) throws ParseException, IOException, SAXException;
+    void parse(InputStream in) throws IOException, SAXException;
```

Comment:
Given that `ParseException` is a checked exception, removing this from the throws clause can cause existing callers of this method (potentially outside of Ivy) to run into compile issues. Is there some specific reason why this is being removed from the throws clause? If not, can you please undo this change.